changelogs/unreleased/gh-8861-feedback-options.md

```diff
@@ -0,0 +1,3 @@
+## feature/config
```

Comment:
Please, rebase after PR #8877 and remove the relevant options in the `test_box_cfg_coverage` test case from the ignore list.

Reply:
Done.